app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -583,14 +586,21 @@ class PipelineSampleReport extends React.Component {
     let includedSubcategories = newIncludedCategories.filter(category => {
       return category in this.category_child_parent;
     });
+
+    const allCategorySet = new Set(
```

Comment:
it seems this block appears twice

Reply:
Indeed, they are local blocks. Again, a little hackish to make it work with old code. Will refactor too with the whole page.